util/file.py

```diff
@@ -397,14 +443,14 @@ def read_tabfile(inFile):
     ''' Read a tab text file (possibly gzipped) and return contents as an
         iterator of arrays.
     '''
-    with open_or_gzopen(inFile, 'rU') as inf:
+    with compressed_open(inFile, 'rt') as inf:
```

Comment:
Since this was originally `rU`, can we have the new invocation include `newline=None` as a kwarg? I think this is trying to defend against the incredibly common scenario of users providing SampleSheets created by MS Excel.